src/core/processing/models/qgsprocessingmodelalgorithm.cpp

```diff
@@ -334,92 +334,6 @@ void QgsProcessingModelAlgorithm::setSourceFilePath( const QString &sourceFile )
   mSourceFile = sourceFile;
 }
 
-QString QgsProcessingModelAlgorithm::asPythonCode() const
```

Comment:
Hmm - we're gonna get flack for removing this. Can we just ifdef it out for now, and then someone can potentially upgrade it to create proper algorithms at some later stage?